distribution/docs/src/main/resources/content/_configuring/importing-exported-configurations-contents.adoc

```diff
@@ -8,18 +8,21 @@
 
 [WARNING]
 ====
-Importing of configurations must take place instead of performing installation for the first time and after performing OS-level hardening.
+Importing of configurations must take place after performing installation for the first time and after performing OS-level hardening.
 ====
 
 To import a previously exported configuration file from the ${command-console}:
 
 . Launch the newly installed ${branding}.
-. Type in `migration:import <directory>`. This command reads the previously exported configuration file `exported-${project-version}.zip` from the specified directory. If no directory is specified it will default to `<${branding}_HOME>/exported`
+. Enter `migration:import <directory>`. This command reads the previously exported configuration file `exported-${project-version}.zip` from the specified directory. If no directory is specified it will default to the `exported` directry underneath `<${branding}_HOME>`.
```

Comment:
The zip isn't really a configuration file. Set of configurations?

Reply:
removed  'configuration'.